x/leverage/client/cli/tx.go

```diff
@@ -198,3 +200,73 @@ func GetCmdRepayAsset() *cobra.Command {
 
 	return cmd
 }
+
+// NewCmdSubmitUpdateRegistryProposal returns a CLI command handler to generate
+// or broadcast a transaction with a governance proposal message containing a
+// UpdateRegistryProposal.
+//
+// NOTE: The "registry" provided in the proposal replaces the entire existing
+// registry.
+func NewCmdSubmitUpdateRegistryProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-registry [proposal-file] [deposit]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Submit a update leverage registry governance proposal",
+		Long: strings.TrimSpace(
+			`Submit a leverage registry governance proposal along with an initial deposit.
+The proposal details must be supplied via a JSON file. Please see the UpdateRegistryProposal
+type for a complete description of the expected input.
+
+Example:
+$ umeed tx gov submit-proposal update-registry </path/to/proposal.json> <deposit> [flags...]
+
+Where proposal.json contains:
+
+{
+  "title": "Update the Leverage Token Registry",
+  "description": "Replace the supported tokens in the leverage registry.",
+  "registry": [
+    {
+      "base_denom": "uumee",
+      "reserve_factor": "40.000000000000000000",
+      "collateral_weight": "43.000000000000000000",
+      "base_borrow_rate": "32.000000000000000000",
+      "kink_borrow_rate": "26.000000000000000000",
+      "max_borrow_rate": "21.000000000000000000",
+      "kink_utilization_rate": "0.250000000000000000",
+      "liquidation_incentive": "88.000000000000000000"
+    },
```

Comment:
```suggestion
    {
      "base_denom": "uumee",
      "reserve_factor": "0.1",
      "collateral_weight": "0.05",
      "base_borrow_rate": "0.02",
      "kink_borrow_rate": "0.2",
      "max_borrow_rate": "1.5",
      "kink_utilization_rate": "0.2",
      "liquidation_incentive": "0.1"
    },
```
better example values